src/main/java/org/launchcode/recipeapp/controllers/IngredientController.java

```diff
@@ -32,7 +32,7 @@ public String displayCreateIngredientForm(Model model) {
         return "ingredients/add";
     }
 
-    @PostMapping("Add")
+    @PostMapping("add")
```

Comment:
I'm not sure this makes a difference, but I always make sure that the capitalization stays the same for things like this.